docs/trex_debugging.md

```diff
@@ -115,22 +114,21 @@ Note that you can only debug one extension, or instance of an extension, at a ti
 
 ---
 
-## Debugging loading and initialization issues (Tableau 2021.1 and later)
+## Debugging loading and initialization issues
+
+### Tableau 2021.1 and later
 
 If you need to troubleshoot or debug issues that prevent your extension from loading or initializing, you can set breakpoints that trigger when your JavaScript code is loaded.
 
 1. Start the debugging session as described in [Debugging Tableau Desktop using Chrome/Chromium](#debugging-tableau-desktop-using-chromechromium).
-
-1. Click the **Sources** tab in Chrome/Chromium, under **Event Listener Breakpoints**, click **Script** and enable the **Script First Statement** breakpoint. You just have to do this one time.
-
-1. In Tableau Desktop, select the extension in the dashboard and click **Reload** from the **More Options** shortcut menu. <br/>
+2. Click the **Sources** tab in Chrome/Chromium, under **Event Listener Breakpoints**, click **Script** and enable the **Script First Statement** breakpoint. You just have to do this one time.
+3. In Tableau Desktop, select the extension in the dashboard and click **Reload** from the **More Options** shortcut menu. <br/>
 The debugger will pause each time the first statement of a script runs, allowing you to debug the startup process.
-
-1. To get to your JavaScript code, click **Continue** several times. After your JavaScript is loaded, you can set a breakpoint in your startup code.
+4. To get to your JavaScript code, click **Continue** several times. After your JavaScript is loaded, you can set a breakpoint in your startup code.
```

Comment:
fyi - In Markdown, if every step is numbered `1`, they automatically get numbered in the correct order when the page is rendered into the GitHub Pages (HTML).  That way you don't have to worry about things when you add or remove steps. I didn't know about this until someone showed me. 

  1. This is numbered `1.`
  1. This is numbered `1.`
  1. This is numbered `1.`

Reply:
Yep - my old habit is to number them so that people reading the text version of the file get a good version too. Here it's probably simpler to number as 1's because people are just reading the generated docs.